.prettierrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "overrides": [
```

Comment:
Fix for a weird issue where the html parser doesn't play nice with `lwc`, but where `lwc` doesn't play nice with non-HTML files.